ci/jenkins/jobs/macros.yaml

```diff
@@ -11,15 +11,34 @@
     builders:
       - shell: |-
           #!/bin/bash
-          set -ex
+
+          saveLogs() {
+            echo "=== Remove too many logs ==="
+            export LOG_DIR=/var/lib/jenkins/antrea_logs
+            find ${LOG_DIR}/* -type d -mmin +10080 | xargs rm -rf
+
+            CLUSTER_LOG_DIR="${LOG_DIR}/${cluster}"
+            echo "=== Saving capi logs ==="
+            mkdir -p ${CLUSTER_LOG_DIR}/capi
+            kubectl get -n capi-system pods -o name | awk '{print $1}' | while read capi_pod; do
+              capi_pod_name=$(echo ${capi_pod} | cut -d'/' -f 2)
+              kubectl logs ${capi_pod_name} -n capi-system --tail=100 > ${CLUSTER_LOG_DIR}/capi/${capi_pod_name}
```

Comment:
I feel `--tail` might be too short. Maybe increase to `--tail=400`?

Reply:
Updated.